src/main/java/com/fasterxml/jackson/databind/EnumNamingStrategies.java

```diff
@@ -0,0 +1,63 @@
+package com.fasterxml.jackson.databind;
+
+public class EnumNamingStrategies {
```

Comment:
For all of these, need to add

```
/**
  * @since 2.15
  */
```

(plus some brief explanation of what class is for etc)